docs/commands/ft.dropindex.md

```diff
@@ -1,41 +1,43 @@
 ---
-syntax: 
+syntax: |
+  FT.DROPINDEX index 
+    [ DD]
```

Comment:
Nitpick - consider a single line for simple commands or multi line - currently it is inconsistent
(some are spanning multi lines and some are not)
```suggestion
  FT.DROPINDEX index 
    [DD]
```